build-test.cmd

```diff
@@ -120,8 +123,8 @@ if defined __BuildAgainstPackagesArg (
     )
 )
 
-echo %__MsgPrefix%Using environment: "%__VSToolsRoot%\VsDevCmd.bat"
-call                                 "%__VSToolsRoot%\VsDevCmd.bat"
+:: echo %__MsgPrefix%Using environment: "%__VSToolsRoot%\VsDevCmd.bat"
+:: call                                 "%__VSToolsRoot%\VsDevCmd.bat"
```

Comment:
Commented out VsDevCmd.bat calls left intententionally to be cleared by #14578

Reply:
Why go ahead and delete them instead of commenting them out?

Reply:
Will fix it